kubernetes-kubelet/tmpfiles.template

```diff
@@ -0,0 +1,3 @@
+d   /etc/cni/net.d - - - - -
```

Comment:
do you want this to be a temporary file?  systemd will destroy/recreate it on reboot.

If you want it to be persistent, you could do:
```
RUN mkdir -p /exports/hostfs/etc/cni/net.d
```
in your Dockerfile

Reply:
I created the dir in the dockerfile, do  I also need to remove it from this list?

Reply:
yes please.  Otherwise systemd-tmpfiles will still try to handle it.